sopel/plugins/handlers.py

```diff
@@ -537,11 +537,11 @@ class EntryPointPlugin(PyModulePlugin):
 
     And this plugin can be loaded with::
 
-        >>> from pkg_resources import iter_entry_points
+        >>> from importlib_metadata import entry_points
```

Comment:
Is that still a "`setuptools` entry point object" in that case? If not, then the docstring above need some adaptation, I think.

Reply:
There are a lot of `setuptools` mentions around the place, honestly. I'm personally not yet clear whether an "entry point" is something that `setuptools` invented, or if it's a Python standard that should be referred to generically. The packaging documentation [seems to hint](https://packaging.python.org/en/latest/specifications/entry-points/) that it's a standard now and we should stop using `setuptools` as a descriptor. What say you?

Also, note to self about docs: there's at least one place that refers to `'sopel.plugins'` as the name of an entry point, rather than a group. Fix that.

Reply:
Oh you're right, it's just "entry point" now. Good!